src/components/head.tsx

```diff
@@ -10,17 +10,18 @@ interface IProps {
 function Head({ setIsSidebarVisible, setSideBarClass }: IProps): JSX.Element {
   const handleSidebar = () => {
     setIsSidebarVisible(true);
-    setSideBarClass('flex flex-col bg-black w-full fixed  h-screen text-white font-roboto justify-between');
+    setSideBarClass('flex flex-col bg-black w-screen fixed  h-screen text-white font-roboto justify-between');
   };
   return (
-    <div className="flex justify-between w-full items-start">
-      <div className="sm:flex w-full justify-between">
-        <h1 className="sm:text-4xl text-3xl pr-16 flex-col justify-between font-bold sm:ml-5">
-          Bonjour, Maxime Savoie
-        </h1>
-        <h2 className="mt-4  text-lg sm:mr-8">{today()}</h2>
+    <div className="flex justify-between w-full h-full items-start">
+      <div className="sm:flex w-full justify-between h-full items-end">
+        <div className="flex-col">
+          <h1 className="text-6xl font-bold">aeviso</h1>
+          <h2 className="text-sm">Expert Comptable.audit.conseil</h2>
+        </div>
+        <h2 className="text-lg mt-4">{today()}</h2>
```

Comment:
Can you just fix this little flaw?


![Capture d’écran 2021-06-20 à 20 51 24](https://user-images.githubusercontent.com/79398461/122685058-87370280-d209-11eb-8ea4-bcb936f1f473.png)